src/Microsoft.Diagnostics.Monitoring.RestServer/MetricsOptions.cs

```diff
@@ -51,5 +52,13 @@ public MetricsOptions()
         public int UpdateIntervalSeconds { get; set; }
 
         public int MetricCount { get; set; }
+
+        public List<MetricProvider> Providers { get; set; } = new List<MetricProvider>(0);
+    }
```

Comment:
Maybe add another option called IncludeDefaultProviders as type bool with a default value of true. Allows customers to exclude the defaults if they want to.